kokoro/builds/build.sh

```diff
@@ -183,7 +183,7 @@ if [ -n "$1" ]; then
 
   # Package the Debian package, the signature and the ggp client into a zip for integration in the 
   # installer.
-  if [ -f ${KEYSTORE_PATH}/74938_SigningPrivateGpg ] && [[ $CONAN_PROFILE == ggp_* ]]; then
+  if true then
```

Comment:
Nit: `if true; then`

But I suggest you leave the `ggp_*` check. Otherwise the windows build will fail.

```
if [[ $CONAN_PROFILE == ggp_* ]]; then
```

Reply:
Ah ok, I realised about the ; but didnt know this. will update it again. Thanks!